src/discord/discord.service.ts

```diff
@@ -1,75 +1,91 @@
-import { Injectable, HttpException, HttpStatus } from '@nestjs/common';
+import { HttpException, HttpStatus } from '@nestjs/common';
+import { createClient } from '@astrajs/collections';
+
 import { ReadDiscordDto } from './dto/read-discord.dto';
 import { UpdateDiscordDto } from './dto/update-discord.dto';
 
-@Injectable()
 export class DiscordService {
-  private discord: ReadDiscordDto[] = [];
-  create(createDiscordDto: ReadDiscordDto) {
+  astraClient;
+  usersCollection;
```

Comment:
Could `astraClient` and `usersCollection` be added to the constructor parameters which then are automatically set as class variables

```ts
constructor(private astraClient: AstraClient, private usersCollection: Collection)
```

Note: check the types, I just took a guess here as an example

Reply:
`@astrajs` is not really designed for TypeScript, so it doesn't have types defined for `astraClient` and `userCollection`. We have to go with any or looking at the returning value of `astraClient` and `userCollection` we need to create types, which I am not sure if they will have the same values all the time or might changes in some scenarios.

@eddiejaoude  Need your suggestion on this.

Reply:
I would us these types for now. It is better than `any`